apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2Test.java

```diff
@@ -116,12 +126,32 @@ public CarrierClaim generateClaim() throws FHIRException {
   public void before() throws IOException {
     when(metricRegistry.timer(any())).thenReturn(metricsTimer);
     when(metricsTimer.time()).thenReturn(metricsTimerContext);
+    npiOrgLookup = Mockito.mockStatic(NPIOrgLookup.class);
```

Comment:
This setup logic is copy/pasted a bunch of times. Should we maybe consider adding a helper utility to centralize some of these things? We already have a utils folder [here](https://github.com/CMSgov/beneficiary-fhir-data/tree/master/apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils)

Reply:
done